pkg/security/probe/probe_windows.go

```diff
@@ -226,6 +226,21 @@ func (p *WindowsProbe) initEtwFIM() error {
 		// try masking on create & create_new_file
 		// given the current requirements, I think we can _probably_ just do create_new_file
 		cfg.MatchAnyKeyword = 0x18A0
+
+		fileIds := []uint16{
```

Comment:
should we make this dynamic ? based on the ruleset ?

Reply:
we could, as a future enhancement.  Right now we're just restricting to the ones we might need, which is a significant win (we don't see `read`s and more, and there's a couple of operations on every file handle we don't see any more either.
But, yes, if we knew from the ruleset there were others we don't want, then we could strip those too.